pkg/dev/podmandev/reconcile.go

```diff
@@ -113,6 +115,42 @@ func (o *DevClient) reconcile(
 		componentStatus.RunExecuted = true
 	}
 
+	// By default, Podman will not forward to container applications listening on the loopback interface.
```

Comment:
Could we make some function with this new code, to try and reduce the lebgth of the reconcile function?

Reply:
Extract this logic into a separate method. See [`17e0292` (#6620)](https://github.com/redhat-developer/odo/pull/6620/commits/17e0292a8210b744532473cb6eeb734090ea0793)